.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest an idea for us to implement!
```

Comment:
"Suggest an idea to improve SunPy".  The current "about" suggests an us/them divide.